.github/workflows/ci-front.yaml

```diff
@@ -41,6 +41,18 @@ jobs:
           fetch-depth: 0
       - name: Install dependencies
         uses: ./.github/workflows/actions/yarn-install
+      - name: GraphQL / Check for Pending Generation
```

Comment:
Let's add this on the server side (ci-server), because this won't run if no file is modified on the frontend. While what should lead a developer to run the metadata generation command is if they've modified something on the server side